apps/web/src/pages/templates/editor/TemplateEditorPage.tsx

```diff
@@ -78,6 +79,14 @@ export default function TemplateEditorPage() {
     }
   }, [template]);
 
+  useEffect(() => {
+    if (environment && template) {
+      if (environment._id !== (template as any)._environmentId) {
```

Comment:
Maybe worth updating the interface of template to include this key instead of the as any here?

Reply:
oh. I did. Sorry. Forgot to remove the any after that